usr/src/cmd/zdb/zdb_il.c

```diff
@@ -43,23 +43,26 @@
 #include <sys/zil_impl.h>
 #include <sys/abd.h>
 
+#include "zdb.h"
+
 extern uint8_t dump_opt[256];
 
 static char prefix[4] = "\t\t\t";
 
 static void
-print_log_bp(const blkptr_t *bp, const char *prefix)
+print_log_bp(const blkptr_t *bp, const char *_prefix)
```

Comment:
leading `_` makes this seem pretty special.  how about changing the global to `tab_prefix`?